components/ws-manager/pkg/manager/manager.go

```diff
@@ -1232,3 +1243,40 @@ func newWssyncConnectionFactory(managerConfig config.Configuration) (grpcpool.Fa
 		return conn, nil
 	}, nil
 }
+
+func checkWSDaemonEntpoint(namespace string, clientset client.Client) func(string) error {
```

Comment:
would a unit test for this function make sense?

Reply:
done